pkg/flagbuilder/build_flags.go

```diff
@@ -176,15 +186,15 @@ func BuildFlags(options interface{}) (string, error) {
 		if flag != "" {
 			flags = append(flags, flag)
 		}
-		// Nothing more to do here
+
 		return utils.SkipReflection
 	}
 	err := utils.ReflectRecursive(reflect.ValueOf(options), walker)
 	if err != nil {
-		return "", err
+		return []string{}, err
```

Comment:
FYI, typically `nil` is basically equivalent to `[]string{}` for all practical use cases, so you can typically just `return nil, err`